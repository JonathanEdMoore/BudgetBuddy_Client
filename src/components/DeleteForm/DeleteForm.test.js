import React from 'react'
import ReactDOM from 'react-dom'
import DeleteForm from './DeleteForm'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe('DeleteForm smoke test', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<DeleteForm />, div)
    ReactDOM.unmountComponentAtNode(div)
  })
})

describe('DeleteForm snapshot test', () => {
  it('renders the UI as expected', () => {
    const wrapper = shallow(<DeleteForm />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})