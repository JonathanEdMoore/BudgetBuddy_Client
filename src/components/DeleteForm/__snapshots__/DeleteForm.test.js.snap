// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DeleteForm snapshot test renders the UI as expected 1`] = `
<div>
  <BudgetBuddyForm
    onSubmit={[Function]}
  >
    <div>
      <label
        htmlFor="email"
      >
        Email
      </label>
      <input
        id="email"
        name="email"
        placeholder="Username"
        type="text"
      />
    </div>
    <div>
      <label
        htmlFor="user_password"
      >
        Password
      </label>
      <input
        id="user_password"
        name="user_password"
        placeholder="password"
        type="password"
      />
      <p
        className="submission-error"
      />
    </div>
    <button
      type="submit"
    >
      Delete Account
    </button>
    <p>
      *Deleting your account will delete all of your data.
    </p>
  </BudgetBuddyForm>
</div>
`;
